static/js/controllers/inbox.js

```diff
@@ -32,6 +33,18 @@ var utils = require('kujua-utils'),
 
       require('../modules/manage-session').init();
 
+      $scope.isDistrict = function(row) {
```

Comment:
Generally I filter before rendering not during (if that makes sense) so I would filter the results before setting on the controller. But if it works it's all good.